scribe/formatted_map.go

```diff
@@ -46,11 +44,16 @@ func (m FormattedMap) String() string {
 
 // NewFormattedMapFromEnvironment take an environment and returns a
 // FormattedMap with the appropriate environment variable information added.
-func NewFormattedMapFromEnvironment(environment packit.Environment) FormattedMap {
+func NewFormattedMapFromEnvironment(environment map[string]string) FormattedMap {
```

Comment:
I see that `packit.Environment` is a type alias for `map[string]string` so this change shouldn't be a breaking change for consumers. But I am curious as to why we want to remove this?

Reply:
See my note above in the PR body. This is so that we can pass both a `packit.Environment` and other string maps (like [the one that gets passed in to `emitter.BuildConfiguration()`](https://github.com/paketo-buildpacks/packit/blob/3a0f37af77a02ea5d2343412614e3a5cfce30f35/scribe/emitter.go#L230)). I figured this would be cleaner than converting the general string map into a `packit.Environment` and then passing that in to the constructor.

Reply:
gotcha - makes sense. I saw the comment in the PR body but I didn't connect it to the explanation in your comment, so thanks for that!